File: ServerSide/Data_Model.py
import json
from json import JSONEncoder
class Authentication:
    def __init__(self, login, password, operation):
        self.login = login
        self.password = password
        self.operation = operation

class AuthenticationEncoder(JSONEncoder):
        def default(self, o):
            return o.__dict__
class Note:
    def __init__(self, name, data, secflag):
        self.name = name
        self.data = data
        self.secflag = secflag
class Notes:
    def __init__(self, notes_):
        self.notes_ = list(notes_)
class NotesEncoder(JSONEncoder):
    def default(self,o):
        return o.__dict__

File: Wpf.BLPmodel.Pages.Core/DataModel.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
namespace ClientSide
{
    public class Data_Model
    {
        public string type;
    }

    [Serializable]
    class Data_Authentication : Data_Model
    {
        public string login;
        public byte[] password;
        public string operation;//авторизация, регистрация, смена пароля
    }
    [Serializable]
    public class Data_Note : Data_Model
    {
        public string name;//Название
        public string data;//содержание
        public int secflag;//уровень допуска

    }
    [Serializable]
    public class Notes_
    {
        public List<Data_Note> notes_ = new List<Data_Note>();
    }
    static class Serialize
    {

        public static string SerializeAuth(Data_Authentication ob)
        {
            return JsonConvert.SerializeObject(ob);
        }
        public static string SerializeNoteRequest(string options, string account)
        {
            return JsonConvert.SerializeObject(options + account);
        }
        public static string SerializeNote(Data_Note data_Note)
        {

            return JsonConvert.SerializeObject(data_Note);
        }
        public static Notes_ DeserializeNote(string jsonstr)
        {
            return JsonConvert.DeserializeObject<Notes_>(jsonstr);
        }
    }
}
